### PR TITLE
Display messages from Failed Queries in UI

### PR DESCRIPTION
**Pull request by miguelknb**
Previously, in the Web UI, when a query fails, both Live Plan and Stage Performance don't show the Query Header. In addition, the same failed query gives no output message in Live Plan.
This change handles an unhandled error inside LivePlan, wich was stopping the render of the Live Plan of a failed query.
It also adds the output message in Live Plan and add the Queary Header in both Live Plan and Stage Performance panels of an failed query.

This Pull Request is the follow up pf #2866

### CODE REVIEW

**Comment by sopel39** [comment]
Could you share screenshots before and after this change?

**Comment by miguelknb** [comment]
Used the query below to generate an error.
```
select (
c.name,
c.address,
c.nationkey,
c.phone,c.name
)
from acustomer as c
```
Heres the live plan and stage performance panels previously:
display-before-liveplan
display-before-stage

And now, here is both of them with the fixes, displaying the Query Header and the right output message:

display-after-liveplan
display-after-stage

File: presto-main/src/main/resources/webapp/src/components/LivePlan.jsx [hunk 1]
```diff
@@ -44,6 +44,9 @@ type StageNodeInfo = {
 
 class StageStatistics extends React.Component<StageStatisticsProps, StageStatisticsState> {
     static getStages(queryInfo): Map<string, StageNodeInfo> {
+        if (!queryInfo.outputStage) {
```
Comment (sopel39) [inline presto-main/src/main/resources/webapp/src/components/LivePlan.jsx:48]: was this a bugfix? If so, please extract to separate commit

File: presto-main/src/main/resources/webapp/src/components/StageDetail.jsx [hunk 2]
```diff
@@ -623,10 +623,13 @@ export class StageDetail extends React.Component {
             );
```
Comment (sopel39) [inline presto-main/src/main/resources/webapp/src/components/StageDetail.jsx:623]: Squash this commit with previous commit

File: presto-main/src/main/resources/webapp/dist/embedded_plan.js [hunk 3]
```diff
@@ -94,7 +94,7 @@
 /***/ (function(module, exports, __webpack_require__) {
```
Comment (sopel39) [inline presto-main/src/main/resources/webapp/dist/embedded_plan.js:94]: suqash this commit

File: presto-main/src/main/resources/webapp/dist/plan.js [hunk 4]
```diff
@@ -94,7 +94,7 @@
 /***/ (function(module, exports, __webpack_require__) {
```
Comment (sopel39) [inline presto-main/src/main/resources/webapp/dist/plan.js:94]: please squash this commit

File: presto-main/src/main/resources/webapp/src/components/LivePlan.jsx (hunk 1 above)
Comment (miguelknb) [reply presto-main/src/main/resources/webapp/src/components/LivePlan.jsx:48]: Yes it was, but it added an error fixed in here https://github.com/prestosql/presto/pull/6214/commits/31f7a7b6095caa1eed2de3937c9250ca12609557

**Comment by sopel39** [comment]
Please squash commits

File: presto-main/src/main/resources/webapp/src/components/LivePlan.jsx [hunk 5]
```diff
@@ -342,6 +348,17 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
             )
         }
 
+        if (query && query.state === "FAILED") {
```
Comment (sopel39) [inline presto-main/src/main/resources/webapp/src/components/LivePlan.jsx:351]: Failed queries contain plan. Does it cause plan to disappear after query failure?

Comment (miguelknb) [reply presto-main/src/main/resources/webapp/src/components/LivePlan.jsx:351]: Until now failed queries plans are not available on the UI, for what I've seen. To display them, I would need to check if they're finished and have an output stage instead of just checking for failed queries.
Something like this:

`if (query && isQueryEnded(query) && query.outputStage ) `

using the utils function isQueryEnded.

Comment (sopel39) [reply presto-main/src/main/resources/webapp/src/components/LivePlan.jsx:351]: > using the utils function isQueryEnded.

That sounds like better idea. Please double check if failed queries do have plan currently (I think they do)

File: presto-main/src/main/resources/webapp/src/components/StageDetail.jsx [hunk 6]
```diff
@@ -623,10 +623,13 @@ export class StageDetail extends React.Component {
...
         }
 
-        if (!this.state.query.outputStage) {
+        if (this.state.query && !this.state.query.outputStage) {
```
Comment (sopel39) [inline presto-main/src/main/resources/webapp/src/components/StageDetail.jsx:626]: was this a bug?

**Comment by bitsondatadev** [comment]
:wave: @miguelknb - this PR is inactive and doesn't seem to be under development. If you'd like to continue work on this at any point in the future, feel free to re-open.
